setup.py

```diff
@@ -28,6 +29,11 @@
 _libtpu_url = (
     f'https://storage.googleapis.com/cloud-tpu-tpuvm-artifacts/wheels/'
     f'libtpu-nightly/libtpu_nightly-0.1.dev{_libtpu_version}-py3-none-any.whl')
+_jaxlib_cuda_url = (
+    f'https://storage.googleapis.com/jax-releases/cuda{{version}}/'
+    f'jaxlib-{_current_jaxlib_version}+cuda{{version}}'
+    f'-cp{sys.version_info.major}{sys.version_info.minor}-none-manylinux2010_x86_64.whl'
+)
```

Comment:
Micro nit: move this in the cuda variables "section" instead of here? Either way though.

Reply:
Done